pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/elastic/ElasticSearchService.java

```diff
@@ -176,6 +179,38 @@ public Dsm getRequiredDsmData(String ddpParticipantId, String index) {
                         ddpParticipantId, index)));
     }
 
+    /**
```

Comment:
All moved from ElasticTestUtil with same signatures, exception for the `Optional<String>` coming back from getParticpantDocumentasString?

Reply:
Pretty much. `_getParticipantDocument` is pretty much the same with a small change.
The other two did not check the response value in ElasticTestUtil (assuming the document was always there).